bundle-size/app.js

```diff
@@ -564,6 +564,60 @@ module.exports = app => {
     }
   });
 
+  v0.post('/commit/:headSha/report.json', async (request, response) => {
+    const {headSha} = request.params;
+    const {baseSha, bundleSizes} = request.body;
+
+    if (typeof baseSha !== 'string' || !/^[0-9a-f]{40}$/.test(baseSha)) {
```

Comment:
Would it make sense to extract this into a `isValidSha` function?

Reply:
I've thought the same thing a few times, I feel like I've seen this in a handful of places in this app (though idr how many were the same N occurrences). Might be worth `isValidSha` or `assertValidSha` that throws

Reply:
Not really, this is only used once, and once again for a soon-to-be-deleted endpoint. I'd rather not extract this into a method in that case :)